src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java

```diff
@@ -111,7 +111,6 @@ public void onPlaced(World world, BlockPos pos, BlockState state, @Nullable Livi
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```

Comment:
Why'd you remove this?

Reply:
There was no deprecation warning with it removed

Reply:
There's a deprecation warning in IntelliJ because you're overriding a deprecated method.